catkin_tools/common.py

```diff
@@ -399,6 +400,19 @@ def log(*args, **kwargs):
             unicode_error_printed = True
 
 
+__warn_terminal_width_once_has_printed = False
+
+
+def __warn_terminal_width_once():
+    global __warn_terminal_width_once_has_printed
+    if __warn_terminal_width_once_has_printed:
+        return
+    __warn_terminal_width_once_has_printed = True
+    print('WARNING: Could not determine the width of the terminal. '
+          'This warning will only be printed once.',
+          file=sys.stderr)
```

Comment:
That's a good idea, I added a note about the default of 80 in the message in 	7d586d4.

I also changed the `WARNING` to `NOTICE`, but that's just to check a hunch about the failing CI.